cdap-common/src/main/java/co/cask/cdap/common/utils/TimeMathParser.java

```diff
@@ -79,20 +83,53 @@ public static int resolutionInSeconds(String resolutionStr) {
     return output;
   }
 
+  /** parses a time in String format into a long value, assuming the input is in seconds if numeric
```

Comment:
Capitalize the first character and you need a full-stop for the first sentence. Also we start javadoc on the second line:

```java
/**
  * Parses ... 
  *
  * @param ...
  */
```